src/aap_eda/core/management/commands/create_initial_data.py

```diff
@@ -48,7 +48,7 @@
 AUTH_TYPE_LABEL = "Event Stream Authentication Type"
 SIGNATURE_ENCODING_LABEL = "Signature Encoding"
 HTTP_HEADER_LABEL = "HTTP Header Key"
-DEPRECATED_CREDENTIAL_KINDS = ["mtls"]
+DEPRECATED_CREDENTIAL_KINDS = [""]
```

Comment:
This should be just an empty string, am I right?

Reply:
It should be `DEPRECATED_CREDENTIAL_KINDS = []`